packages/schema/src/interfaces.ts

```diff
@@ -78,6 +78,7 @@ export interface IJupyterCadDoc extends YDocument<IJupyterCadDocChange> {
   objects: Array<IJCadObject>;
   options: JSONObject;
   metadata: JSONObject;
+  editable: boolean;
```

Comment:
```suggestion
  readonly editable: boolean;
```
Should it be read-only?